src/lib/slide-toggle/slide-toggle.scss

```diff
@@ -24,6 +24,12 @@ $md-slide-toggle-margin: 16px !default;
   line-height: $md-slide-toggle-height;
 
   white-space: nowrap;
+
+  // Disable user selection to ensure that dragging is smooth without grabbing some elements
+  // accidentally. Manually prefixing here, because the un-prefixed property is not supported yet.
+  -webkit-user-select: none;
+  -moz-user-select: none;
+  -ms-user-select: none;

```

Comment:
Seems like there is no autoprefixer anymore, and `user-select` is also kind of a special property for the prefixer (e.g Material 1)